enable/tests/tools/test_hover_tool.py

```diff
@@ -53,7 +53,7 @@ class GuiTestAssistant(object):
 class HoverToolTestCase(EnableTestAssistant, GuiTestAssistant,
                         unittest.TestCase):
     def setUp(self):
-        super(HoverToolTestCase, self).setUp()
+        GuiTestAssistant.setUp(self)
```

Comment:
Same question I had on the other PR, why exactly do we purposefully avoid super for setUp

Reply:
answered elsewhere - part convention, part best practice - especially because we have cases where we need to call `setUp` on more than one base class.